### ISSUE TITLE
Windows Defender detected Trojan:Win32/Sabsik.FL.A!ml in fabric

### ISSUE THREAD

**Issue by NicoFlows** [danielmiessler/Fabric#1051]
Published Apr 28, 2021 | Updated Not applicable
Learn about other threats
Trojan:Win32/Sabsik.FL.A!ml
[Detected by Microsoft Defender Antivirus](https://www.microsoft.com/en-us/windows/windows-defender?ocid=cx-wdsi-ency)

Aliases: No associated aliases

Windows 11 
Using the -> https://github.com/danielmiessler/fabric/releases/download/v1.4.65/fabric-windows-amd64.exe

I had fabric working and was looking forward to using it. However I don't want to restore it until you have had a chance to check on this.

Thanks

**Comment by ksylvan** [comment]
This is definitely a false positive and I believe it has been resolved.

@tonymet do you want to comment on this?

**Comment by tonymet** [comment]
@NicoFlows I observed a similar report of `Trojan:Win32/Wacatac.C!ml` and believe it's a false positive due to lack of code signing.  Could you share the steps you used to download? was it via browser (which browser) or shell command? 

I recorded the full MS Defender Diagnostic report and steps to repro here, but I've been having inconsistent alerts from MS Defender:

https://gist.github.com/tonymet/b920bbd93d06e4082f9343a5c6fc00eb

**Comment by tonymet** [comment]
@NicoFlows  Please use this powershell command to share the full event message contents from MS Defender

```
Get-WinEvent -FilterHashtable @{
    LogName = 'Microsoft-Windows-Windows Defender/Operational'
    ID      = 1116, 1117
} | Format-List
```

**Comment by tonymet** [comment]
> This is definitely a false positive and I believe it has been resolved.

@ksylvan I agree it's likely a false positive, but we should track this somewhere, possibly privately until we have a complete fix for the false positives.  Looks like this report is a slightly different alarm than mine.  Likely both are false positives due to heuristics.

**Comment by tonymet** [comment]
## My current theories
1.  a user accidentally reported this URL (because the download prompt looks scary on MS Edge )
2.  behavioral heuristics are being triggered by local writes, network activity (less commonly, memory access)
3. lack of code signing -- but few open source apps are code-signing.

**Comment by tonymet** [comment]
@NicoFlows you can start a custom MS Defender Scan with this command.  See if that continues to trigger the alarm

```
Start-MpScan -ScanType CustomScan -Scanpath (pwd)
```

**Comment by tonymet** [comment]
@NicoFlows  could you test installing via winget to see if your MS Defender is triggered

```
 winget install danielmiessler.Fabric
```
